### DISCUSSION TITLE
The color of WebSocket Visualizer status bar is blue

### DISCUSSION THREAD

**Discussion by yichengzhizhu007** [eclipse/mosaic#70] [Q&A]
When I start the visualization of the tutorial scenario Barnim，the color of WebSocket Visualizer status bar is blue. I can't see vehicle markers moving around and indicating if they are sending V2X messages (green) or receiving V2X message (red).why？PS：My operating system is windows 10 x64
![6FBD90E2-833A-4f45-88B3-37B9DFFEBF1E](https://user-images.githubusercontent.com/66045520/112408808-67e82080-8d53-11eb-9201-9902fff1409d.png)

**Comment by realmaxneu** [comment]
Did you install Eclipse SUMO on your system? It is required for most tutorials, also for Barnim, and is not installed automatically along Eclipse MOSAIC. You need to do that separately.
Did you run Eclipse Mosaic with the parameter `--realtime-brake`? You can find more details about that in our [documentation](https://www.eclipse.org/mosaic/docs/getting_started/run_mosaic/#cli-options). Please try `./mosaic.sh -s Barnim -b 1`.

**Comment by DemingCheng** [comment]
If you are in China, maybe you should use the VPN. I resolve the same problem with the help of VPN although I don‘t konw why it is essential.

**Reply by kschrab** [reply]
I would assume that some of the JS libraries required by the WebSocket Visualizer are blocked in China and therefore it would only work using VPN.

**Reply by yichengzhizhu007** [reply]
The problem is solved by using the VPN~thank you~

**Reply by DemingCheng** [reply]
Do you have the QQ or wechat? would you like to  discuss somenthing about this tool?
